.travis.yml

```diff
@@ -24,7 +24,7 @@ matrix:
       env: BUILD_DOCS=true
 
 install:
-  - pip install -q --use-mirrors nose python-dateutil $NUMPY pep8 pyparsing pillow sphinx!=1.3.0
+  - pip install python-dateutil $NUMPY pep8 pyparsing pillow sphinx!=1.3.0 coveralls coverage
```

Comment:
Yes, nose is installed by default on travis, I was looking for version conflict issues (there were a whole bunch of force-pushed attempts at this).